packages/react-table/src/docs/demos/table-demos/StaticBottomPagination.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import {
+  Button,
+  Card,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+  Pagination,
+  Select,
+  SelectVariant,
+  SelectOption,
+  PageSection
+} from '@patternfly/react-core';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import { rows, columns } from '../../examples/Data.jsx';
+
+import FilterIcon from '@patternfly/react-icons/dist/esm/icons/filter-icon';
+import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
+
+export const StaticBottomPagination = () => {
+  const [isSelectOpen, setIsSelectOpen] = React.useState(false);
+  const [page, setPage] = React.useState(1);
+  const [perPage, setPerPage] = React.useState(10);
+  const [paginatedRows, setPaginatedRows] = React.useState(rows.slice(0, 10));
+  const handleSetPage = (_evt, newPage, perPage, startIdx, endIdx) => {
+    setPaginatedRows(rows.slice(startIdx, endIdx));
+    setPage(newPage);
+  };
+  handlePerPageSelect = (_evt, newPerPage, newPage, startIdx, endIdx) => {
+    setPaginatedRows(rows.slice(startIdx, endIdx));
+    setPage(newPage);
+    setPerPage(newPerPage);
+  };
+
+
+  const renderPagination = (variant, isCompact) => (
+    <Pagination
+      isCompact={isCompact}
+      itemCount={rows.length}
+      page={page}
+      perPage={perPage}
+      onSetPage={handleSetPage}
+      onPerPageSelect={handlePerPageSelect}
+      variant={variant}
+      titles={{
+        paginationTitle: `${variant} pagination`
+      }}
+    />
+  );
+
+  const renderLabel = labelText => {
+    switch (labelText) {
+      case 'Running':
+        return <Label color="green">{labelText}</Label>;
+      case 'Stopped':
+        return <Label color="orange">{labelText}</Label>;
+      case 'Needs Maintenance':
+        return <Label color="blue">{labelText}</Label>;
+      case 'Down':
+        return <Label color="red">{labelText}</Label>;
+    }
+  };
+
+  const tableToolbar = (
+    <Toolbar usePageInsets id="compact-toolbar">
```

Comment:
Nice catch! @kmcfaul did it right, the core demo needs to be updated. We may have left off the inset because the default inset magically aligns these checkboxes 

<img width="1572" alt="Screen Shot 2022-09-02 at 1 32 43 PM" src="https://user-images.githubusercontent.com/35148959/188216569-8ac83ad9-0b43-4e74-955b-d75cb758527d.png">

@mcarrano should we remove that bulk select in the toolbar and move it to the table header? Like this example - https://www.patternfly.org/v4/components/table/html#checkboxes-and-actions-example

Reply:
Actually, looking at the [react bulk select demo](https://patternfly-react-pr-7896.surge.sh/components/table/react-demos/bulk-select/), that toolbar isn't using page insets, either.